googleapis/google-cloud-node
# migrate code from googleapis/nodejs-api-gateway

- chore: add initial files for launch
- chore: add initial files for launch
- feat!: initial generation of library (#1)
- chore: release 1.0.0 (#3)
- build: update linkinator, copyright year (#5)
- chore: add gapic build rules (#9)
- chore(apigateway): Update namespaces for Ruby clients (#10)
- chore: migrate to owl bot (#11)
- chore(deps): update dependency sinon to v10 (#15)
- chore(deps): update dependency ts-loader to v9 (#19)
- fix(deps): require google-gax v2.12.0 (#23)
- chore: new owl bot post processor docker image (#25)
- fix: use require() to load JSON protos (#26)
- chore: update gapic-generator-typescript to v1.3.2 (#27)
- chore: release 1.0.1 (#24)
- feat: bump release level to GA (#30)
- chore: release 1.1.0 (#31)
- chore(deps): update dependency sinon to v11 (#32)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#33)
- chore: release 1.1.1 (#34)
- fix: make request optional in all cases (#40)
- fix(deps): google-gax v2.17.0 with mTLS (#44)
- chore: release 1.1.2 (#41)
- fix(deps): google-gax v2.17.1 (#46)
- chore: release 1.1.3 (#47)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#48)
- chore: release 1.1.4 (#49)
- chore(nodejs): update client ref docs link in metadata (#54)
- fix(build): migrate to using main branch (#56)
- chore: release 1.1.5 (#57)
- fix(deps): google-gax v2.24.1 (#58)
- chore: release 1.1.6 (#59)
- feat: turns on self-signed JWT feature flag (#60)
- chore: release 1.2.0 (#61)
- build(node): update deps used during postprocessing (#1243) (#70)
- chore(cloud-rad): delete api-extractor config (#71)
- chore: fix the wrong post processor image (#73)
- chore(deps): update dependency @types/node to v16 (#72)
- chore(deps): update dependency sinon to v12 (#74)
- docs(samples): add example tags to generated samples (#75)
- build: add generated samples to .eslintignore (#76)
- docs(node): support "stable"/"preview" release level (#1312) (#79)
- chore: add api_shortname and library_type to repo metadata (#78)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#81)
- test(nodejs): remove 15 add 16 (#1322) (#83)
- fix(apigateway): add ancillary service bindings to service_yaml (#84)
- chore(main): release 1.2.1 (#85)
- chore: update v2.12.0 gapic-generator-typescript (#88)
- chore(deps): update dependency sinon to v13 (#90)
- docs(nodejs): version support policy edits (#1346) (#92)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#93)
- chore: update v2.14.2 gapic-generator-typescript (#97)
- build(node): update client library version in samples metadata (#1356) (#104)
- chore(deps): update dependency @types/mocha to v9 (#105)
- chore(deps): update dependency sinon to v14 (#108)
- build!: update library to use Node 12 (#109)
- chore(main): release 2.0.0 (#111)
- chore(deps): update dependency jsdoc-fresh to v2 (#114)
- chore(deps): update dependency jsdoc-region-tag to v2 (#115)
- feat: support regapic LRO (#116)
- chore(main): release 2.1.0 (#117)
- chore(deps): update dependency linkinator to v4 (#121)
- chore: remove unused proto imports (#122)
- fix: better support for fallback mode (#123)
- fix: change import long to require (#124)
- chore(main): release 2.1.1 (#125)
- fix: do not import the whole google-gax from proto JS (#1553) (#127)
- fix: allow passing gax instance to client constructor (#128)
- fix: preserve default values in x-goog-request-params header (#130)
- test: use fully qualified request type name in tests (#132)
- fix(deps): use google-gax v3.5.2 (#136)
- chore(deps): update dependency @types/node to v18 (#134)
- chore(deps): update dependency jsdoc to v4 (#138)
- fix: regenerated protos JS and TS definitions (#139)
- chore(main): release 2.1.2 (#129)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
